validation/validation.go

```diff
@@ -47,13 +53,21 @@ func validateIssueTracker(cfg *config.Local) error {
 }
 
 func validateAuthOfflineURL(cfg *config.Local) error {
-	if _, err := url.ParseRequestURI(cfg.Auth.OfflineURL); cfg.Auth.Type == config.AuthTypeOffline && err != nil {
+	if _, err := url.ParseRequestURI(cfg.Auth.OfflineURL); err != nil {
 		return fmt.Errorf("invalid offline URL: %q", cfg.Auth.OfflineURL)
 	}
 
 	return nil
 }
 
+func validateAuthOfflineUrlIsSet(cfg *config.Local) error {
```

Comment:
Please capitalize `URL`. Also, consider setting up `gofmt` as it will probably show a warning in this case.

Reply:
Good point! change is now implemented.